cpp/include/kvikio/posix_io.hpp

```diff
@@ -41,17 +41,8 @@ class StreamsByThread {
   std::map<std::pair<CUcontext, std::thread::id>, CUstream> _streams;
 
  public:
-  StreamsByThread() = default;
-  ~StreamsByThread() noexcept
-  {
-    for (auto& [_, stream] : _streams) {
-      try {
-        CUDA_DRIVER_TRY(cudaAPI::instance().StreamDestroy(stream));
-      } catch (const CUfileException& e) {
-        std::cerr << e.what() << std::endl;
-      }
-    }
-  }
+  StreamsByThread()  = default;
+  ~StreamsByThread() = default;
```

Comment:
It would be good with a comment and an URL to the issue in the CUDA programming guide

Reply:
Good idea! I saw other source files in kvikio doing the same thing. I'll follow suit.

Reply:
Completed.